Section 4 - Terraform Modules & Workspaces/module-locals/modules/sg/sg.tf

```diff
@@ -1,39 +1,39 @@
 resource "aws_security_group" "ec2-sg" {
-  name        = "myec2-sg"
+  name = "myec2-sg"
 
   ingress {
-    description      = "Allow Inbound from Secret Application"
-    from_port        = 8433
-    to_port          = 8433
-    protocol         = "tcp"
-    cidr_blocks      = ["0.0.0.0/0"]
+    description = "Allow Inbound from Secret Application"
+    from_port   = 8433
+    to_port     = 8433
+    protocol    = "tcp"
+    cidr_blocks = ["<cidr>"]
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure Unknown Port is not exposed to the entire internet**
 
**Vulnerability**
Unknown Port is exposed to the entire internet. Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: tenablecs-